test/hook/v16-declaration-exports.mjs

```diff
@@ -0,0 +1,54 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2.0 License.
```

Comment:
This, and all other test files here, only test from v16 and up. What's the reason for this?

Reply:
Alot of the new tests contain syntax that only works in node >= 16